src/main/java/com/example/pfairplayservice/model/Member.java

```diff
@@ -44,18 +54,36 @@
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     private Date recentLoginDate;
 
-    public static Member from(MemberEntity member) {
+    public static Member from(MemberEntity memberEntity) {
 
-        Member result = new Member(member.getUid(), member.getName(), member.getId(), member.getPassword()
-                , member.getBirthday(), member.getAddress(), member.getPhoneNumber(), Position.from(member.getPreferPosition())
-                , member.getLevel(), member.getPhoneNumberDisclosureOption(), member.getJoinDate(), member.getRecentLoginDate());
+        Member result = new Member(memberEntity.getUid(), memberEntity.getName(), memberEntity.getId(), Team.fromList(memberEntity.getTeamEntityList()), memberEntity.getPassword()
+                , memberEntity.getBirthday(), memberEntity.getAddress(), memberEntity.getPhoneNumber(), Position.from(memberEntity.getPreferPosition())
+                , memberEntity.getLevel(), memberEntity.getPhoneNumberDisclosureOption(), memberEntity.getJoinDate(), memberEntity.getRecentLoginDate());
```

Comment:
Please consider the builder pattern for creating new object and set each fields.
https://en.wikipedia.org/wiki/Builder_pattern
https://projectlombok.org/features/Builder

Reply:
finish applying builder pattern